docs/predictive-analytics/model-retraining/manual.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO predicted_weight VALUES ('101', 'Salmon', 17.33, 74.55, 3.78);
+INSERT INTO predicted_weight VALUES ('102', 'Salmon', 19.11, 82.19, 4.17);
+INSERT INTO predicted_weight VALUES ('103', 'Salmon', 21.07, 90.62, 4.6);
+INSERT INTO predicted_weight VALUES ('104', 'Bass', 15.44, 56.23, 2.54);
+INSERT INTO predicted_weight VALUES ('105', 'Bass', 17.02, 62, 2.8);
+INSERT INTO predicted_weight VALUES ('106', 'Bass', 18.76, 68.34, 3.09);
+INSERT INTO predicted_weight VALUES ('107', 'Trout', 13.34, 64.05, 1.47);
+INSERT INTO predicted_weight VALUES ('108', 'Trout', 14.71, 70.61, 1.62);
+INSERT INTO predicted_weight VALUES ('109', 'Trout', 16.22, 77.85, 1.79);
+INSERT INTO predicted_weight VALUES ('110', 'Trout', 17.03, 81.74, 1.88);
+INSERT INTO actual_weight VALUES ('101', 'Salmon', 4.38);
```

Comment:
nit: insert blank line to separate inserts into both STREAMs?

Reply:
Done